### ISSUE TITLE
Can't find save block on digital Scarlet + DLC

### ISSUE THREAD

**Issue by Eppin** [LegoFigure11/RaidCrawler#135]
**Describe the bug**
Hi, I'm not really sure if this a bug or not, but also not really sure where to put this.

**To Reproduce**
I own two CFW console with and I've tried quite a lot of different setups:
CFW Switch 1:
- EmuNAND:
-- Dumped Scarlet (without DLC): works, can connect, view raids and progress is set
- SysNAND:
-- Digital Scarlet (with DLC bought): doesn't work, execption is thrown
-- Physical Violet (without DLC): works, can connect, view raids and progress is set

CFW Switch 2:
- SysNAND:
-- Physical Scarlet: works, can connect, view raids and progress is set

**Expected behavior**
Be able to connect successfully, progress is read and raids.

**Screenshots**
I don't have a screenshot, however I did some debugging and it fails here: https://github.com/LegoFigure11/RaidCrawler/blob/main/MainWindow.cs#L313. It seems `if (read_key == key)` will never be true. I've also tried searching through the RAM, but wasn't been able to find the keys at different offsets and I've also restarted the Switch multiple times, tried different sys-botbase, but the weirdest part is that it seems to only affect the digital Scarlet, because physical Violet works perfectly.

Do you have any clue?

**Comment by ruanick** [comment]
I was having the same error message when using a physical copy of Violet without DLC. This was the case when using either the latest release download or the nightly link from the PKHeX Discord server. 

Then I cloned the repository and ran the program in Visual Studio and it connected successfully. Surprisingly now I can just run the nightly version without issue. A warning to anyone thinking about trying this, I ran an unreleased version of the software and accepted that it could come with side effects.

I think it might be related to the fact that I had never connected my game or console to this software before. Somehow running it via Visual Studio allowed that first ever connection and now the nightly version is able to connect.

It was also my first time ever running and connecting successfully with the software, maybe Visual Studio provided some dependency that I was previously missing. I don't have a lot of experience with C# and Visual Studio wasn't previously installed on my computer.

Not sure what the program status said at the time of crashing, I wasn't expected the issue to be suddenly resolved so I hadn't written it down. But I think it was during "Getting Story Progress". Maybe someone else experiencing the issue can confirm.
